libethereum/BlockQueue.cpp

```diff
@@ -131,6 +127,7 @@ void BlockQueue::verifierBody()
 				}
 			cwarn << "BlockQueue missing our job: was there a GM?";
 			OK1:;
+			drainVerified();
```

Comment:
Here if bad block is at m_verifying[0] and m_verifying[1] contains a verified block latter will be never removed from m_verifying blocking the queue.